sickbeard/subtitles.py

```diff
@@ -849,6 +849,10 @@ def dhm(td):
                              ep_to_sub['show_name'], ep_num)
                 continue
 
+            if sickbeard.SUBTITLES_STOP_AT_FIRST and ep_to_sub['subtitles']:
```

Comment:
Yeah, I just used it as an example.

After you have 1 sub, stop looking for more, since you're fine.